eng/tools/azgoperf/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Why isn't this MIT?

Reply:
This was the default by cobra's cli. Working on cleaning a few things up still, this is on that list.